index.test.js

```diff
@@ -35,13 +35,52 @@ const Hr = ['hr']
 
 export default {
   Element: {
-    base: [hh(Button), { x: 'button', y: { onclick: noop }, z: 'testing' }],
-    'no children': [hh(Input), { x: 'input', y: { type: 'range' }, z: [] }],
-    'no props': [hh(Title), { x: 'h1', y: {}, z: 'testing' }],
-    'no props, no children': [hh(Hr), { x: 'hr', y: {}, z: [] }],
-    'children 0': [hh(Zero), { x: 'p', y: {}, z: '0' }],
-    'just string': [hh('Hello'), 'Hello'],
+    base: [
```

Comment:
just expanding the lines for readability

Reply:
Prettier compacts these for me, but I am going to turn off prettier in tests I think 😅